getting-started/templates/systemlink-values.yaml

```diff
@@ -503,6 +503,43 @@ dataframeservice:
       ## The name of the key in the above secret whose value contains the Dremio password
       ##
       passwordKey: "password"
+    distStorage:
+      # Dremio distributed storage configuration. This must be configured for the service to perform acceptably.
+      # See https://github.com/dremio/dremio-cloud-tools/blob/master/charts/dremio_v2/docs/Values-Reference.md#distributed-storage-values
+      # <ATTENTION>: This only partially configures distributed storage. Credentials must be configured in systemlink-secrets.yaml under
+      # sldremio.distStorage.aws.credentials.
+      type: "aws" # <ATTENTION> - change this if not using Amazon S3 or an equivalent, like MinIO
+      aws:
+        bucketName: *dataframeBucket
+        path: "/dremio/distStorage"
+        authentication: "accessKeySecret"
+        # <ATTENTION> - Configure credentials in systemlink-secrets.yaml under sldremio.distStorage.aws.credentials
```

Comment:
I'm not sure it's useful to double-up <ATTENTION> tags between both files.

Reply:
Removed it.